### PR TITLE
#5194 [Manager] Support nested JSON format of topic data in developer…

### PR DESCRIPTION
**Pull request by eechih**
… tools

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #5194 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js [hunk 1]
```diff
@@ -72,7 +72,11 @@ const ViewTopic = () => {
                 const align = headerIdx === lastItem ? 'right' : 'left';
                 return message.value[header] ? (
                   <StyledTableCell align={align} key={rowIdx + '_' + headerIdx}>
-                    {message.value[header]}
+                    {isEmpty(message.value[header])
+                      ? null
```
Comment (eechih) [inline ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js:76]: 避免顯示 `{}` `[]`

File: ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js [hunk 2]
```diff
@@ -72,7 +72,11 @@ const ViewTopic = () => {
                 const align = headerIdx === lastItem ? 'right' : 'left';
                 return message.value[header] ? (
                   <StyledTableCell align={align} key={rowIdx + '_' + headerIdx}>
-                    {message.value[header]}
+                    {isEmpty(message.value[header])
```
Comment (saivirtue) [inline ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js:75]: is this line duplicated to Line:73 ?

Comment (eechih) [reply ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js]: I didn't notice, fix it immediately :)

File: ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js [hunk 3]
```diff
@@ -72,7 +72,11 @@ const ViewTopic = () => {
                 const align = headerIdx === lastItem ? 'right' : 'left';
                 return message.value[header] ? (
                   <StyledTableCell align={align} key={rowIdx + '_' + headerIdx}>
-                    {message.value[header]}
+                    {isObjectLike(message.value[header])
+                      ? isEmpty(message.value[header])
+                        ? null
```
Comment (eechih) [inline ohara-manager/client/src/components/DevTool/Body/View/ViewTopic.js:77]: 避免顯示空物件或空陣列 `{} []`

**Comment by eechih** [comment]
retry manager-e2e

**Pull request merged**